pkgs/development/python-modules/python-pam/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, buildPythonPackage, fetchPypi, pam }:
+
+buildPythonPackage rec {
+  pname = "python-pam";
+  version = "1.8.4";
+  
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c856d9c89fedb33951dd8a95727ae57c6887b02d065bbdffd2fd9dbc0183909b";
+  };
+  
+  # ctypes.util.find_library does not work on Nix
+  patchPhase = ''
+    substituteInPlace pam.py --replace 'find_library("pam")' '"${pam.out}/lib/libpam${stdenv.hostPlatform.extensions.sharedLibrary}"'
+  '';
+  
+  # there is no test code for this module
+  doCheck = false;
+  
```

in lieu of tests, please at least do
```suggestion
  pythonImportsCheck = [ "pam" ];
```